extensions/wikia/AdEngine/src/ad-engine.bridge.js

```diff
@@ -0,0 +1,49 @@
+import Context from 'ad-engine/src/services/context-service';
+import SlotService from 'ad-engine/src/services/slot-service';
+import TemplateService from 'ad-engine/src/services/template-service';
+import BigFancyAdBelow from 'ad-products/src/modules/templates/uap/big-fancy-ad-below';
+import UniversalAdPackage from 'ad-products/src/modules/templates/uap/universal-ad-package';
+import config from './context';
+
+let blbSlot = {
+	config: {
+		targeting: {}
+	},
+	getSlotName: () => 'BOTTOM_LEADERBOARD',
+	getId: () => 'BOTTOM_LEADERBOARD'
+};
+
+let supportedTemplates = [ BigFancyAdBelow ];
+
+Context.extend(config);
+
+supportedTemplates.forEach((template) => {
+	TemplateService.register(template);
+});
+
+SlotService.add(blbSlot);
+
+function getSupportedTemplateNames() {
+	return supportedTemplates.map((template) => template.getName());
+}
+
+function loadCustomAd(fallback) {
+	return (params) => {
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Convert this ES6 arrow function into a regular one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-js-rules%3AArrowFunctions)

[project-id]: wikia:app